### PR TITLE
Add warning for operator name

### PR DESCRIPTION
**Pull request by adamdossa**

### CODE REVIEW

**Review by maxsam4 — APPROVED** [review]

File: docs/operator/README.md [hunk 1]
```diff
@@ -415,6 +415,8 @@ To run a polymesh node we recommend that you make use of the following options:
   in-memory cache above the default `128` MiB.  On a node with 8GB RAM available a reasonable value is in the
   ballpark of `4096`.
 
+Note - the (optional) `<name>` above will be public at:  
+https://stats.polymesh.live/
```
Comment (cwsatpolymath) [inline docs/operator/README.md:419]: Do we want to include the stats endpoint here, or just rephrase this to e.g. 

> Note - the (optional) `<name>` parameter above will be publicly visible if sending telemetry to Polymath's servers is enabled (on by default).

**Review by cwsatpolymath — CHANGES_REQUESTED** [review]
